python-client/giskard/push/utils.py

```diff
@@ -57,10 +57,37 @@ def slice_bounds(feature: str, value: Union[int, float], ds: Dataset) -> Optiona
             return [q1, q2]
         elif q2 <= value < q3:
             return [q2, q3]
-        else:
-            return [q3, ds.df[feature].max()]
-    else:
-        return None
+        return [q3, ds.df[feature].max()]
+    return None
+
+
+def slice_bounds_relative(
+    feature: str, value: Union[int, float], ds: Dataset, window_size: float = 0.1
+) -> Optional[List[Union[int, float]]]:
+    """Get fixed bounds values to slice Giskard dataset on a numerical feature.
+
+    Args:
+        feature (str): Feature name
+        value (float or int): Feature value
+        ds (Dataset): Dataset
+        window_size (float): interval in percentage around the value
+
+    Returns:
+        list: Lower and upper bounds of the slice
+    """
+    if ds.column_types[feature] == "numeric":
```

Comment:
should the condition be like in a caller method (contribution.py): ` != "text"` to be consistent?

Reply:
The bounds are solely for numeric features, not categorical ones